### DISCUSSION TITLE
TuringCourses/major_elective/ai_logic/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#78] [General]
# TuringCourses/major_elective/ai_logic/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_elective/ai_logic/

**Comment by DaViD-Pigeon** [comment]
这里也贴一个cxyjj人工智能逻辑课程的笔记：https://tekhnec.github.io/categories/，好吧这是cxyjj博客的链接hhh

**Comment by crd2333** [comment]
贴一个23-24春夏的回忆卷 https://www.cc98.org/topic/5926692
总体上而言确实跟平时小测的难度差不多，但是考题覆盖面很全，要注意可能会多一些小测没考过的东西，比如：论证攻击关系先分解后合成的求外延方法、抽象论辩框架怎么得到论证攻击关系（这两块我印象里上课讲的时候就比较含糊，但依旧会考）

**Comment by dieKarotte** [comment]
这门课的要求比较高，包括课程、小测、作业，ai专业的同学在大二下普遍要比cs专业的同学多1-2门专业课，可能没有比较多的时间分给这门课程，笔者上课也主打了一个陪伴，并在课后2倍速观看回放，很感谢lbs老师清晰的录播。lbs老师同时开设了哲学学院的数理逻辑，和这门课有较多重合，可以选修作为跨专业模块课程。如果考虑大二下的压力，并要满足学分要求，可以选择大三上thj老师的脑启发，这门课程也是3学分，会相对轻松一些。

**Comment by shrike-505** [comment]
老师期末考试的分数十分的求是，基本你写了多少分的内容就会给多少分然后算进总评[ac01]
（样本来自我和几个同学）
